x-pack/plugins/serverless_search/public/application/components/overview.tsx

```diff
@@ -82,13 +84,17 @@ export const ElasticsearchOverview = () => {
     () => (consolePlugin?.EmbeddableConsole ? <consolePlugin.EmbeddableConsole /> : null),
     [consolePlugin]
   );
+  const [selectedPipeline, setSelectedPipeline] = React.useState<string>('');
 
   const codeSnippetArguments: LanguageDefinitionSnippetArguments = {
     url: elasticsearchURL,
     apiKey: clientApiKey,
     cloudId,
+    ingestPipeline: selectedPipeline,
   };
 
+  const { data: pipelineData } = useIngestPipelines();
```

Comment:
Did you test negative cases for this call? we're not checking for an error so I assume if this fails then there are just no pipelines available?

Reply:
If there is no pipelines available, the combo-box provide an empty [list of options](https://github.com/elastic/kibana/pull/182604/files#diff-0ea2e0424b5820da77b207323edbe66258d9e19a482ab0fd714b97e074384a50R102). I will add additional tests for this scenario.